drizzle-orm/src/pg-core/README.md

```diff
@@ -224,7 +224,7 @@ export const cities = pgTable('cities', {
   id: serial('id').primaryKey(),
   name: varchar('name', { length: 256 }),
   countryId: integer('country_id').references(() => countries.id), // inline foreign key
-  countryName: varchar('country_id'),
+  countryName: varchar('country_name'),

```

Comment:
Does this need a `.references(() => countries.name)` or is that redundant with line 238?

Reply:
yup, redundant